internal/supervisor/installation.go

```diff
@@ -593,6 +647,43 @@ func (s *InstallationSupervisor) waitForUpdateStable(installation *model.Install
 	return model.InstallationStateStable
 }
 
+// Unused stub function
```

Comment:
curious, why keep this if it is unused?

Reply:
Sorry, I mentioned it in the PR description, but it will be used shortly and don't want it to become stale.